readthedocs/starterguides/installerguides/LocalClusterAnsibleInstallerUserGuide.md

```diff
@@ -0,0 +1,114 @@
+This document describe how to install and use OpenSDS projects by using Ansible Installer. 
+This document builds upon installer wiki https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible
+## Installation requirements
+* Ubuntu 16.04 and packages
+* Docker-ce 18.06
+* Ansible 2.4.6
+Please refer [wiki page](https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible) to install above requirements.
+## Install OpenSDS using opensds-installer
+Clone opensds-installer repository and checkout Capri tag.
+```bash
+git clone https://github.com/opensds/opensds-installer.git
+cd opensds-installer
+git checkout v0.6.0
+```
+### Default installation
+Below command will install default OpenSDS configuration and its dependant components in the Host/VM, where installer in running and host_ip set to localhost (127.0.0.1).
+```bash
+cd ansible
+ansible-playbook site.yml -i local.hosts 
+```
+Using OpenSDS
+How to test opensds cluster
+### Specific feature configurations and installation
+Installer may be configured for specific features using configurations from YML files in path `ansible/group_vars/`.
+Following are some of the configurations that may be useful. More details available in [wiki page](https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible)
+#### Common
+Specific host: `host_ip` in `ansible/group_vars/common.yml`
+```yml
+# This field indicates 
+host_ip: 127.0.0.1
+```
+#### Authentication
+Authentication strategy: `opensds_auth_strategy` in `ansible/group_vars/auth.yml`
+```yml
+# OpenSDS authentication strategy, support 'noauth' and 'keystone'.
+opensds_auth_strategy: keystone
+```
+#### Telemetry
+Enable Telemetry: `enable_telemetry_tools` in `ansible/group_vars/telemetry.yml`
+```yml
+# Do you need to install or clean up telemetry tools?
+enable_telemetry_tools: true 
+```
+#### Orchestration
+Enable Orchestration: `enable_orchestration` in `ansible/group_vars/orchestration.yml`
+```yml
+# Install Orchestration Manager (true/false)
+enable_orchestration: true 
+```
+### Un-install and Clean-up
+Below command will clean OpenSDS and its dependant components
+```bash
+ansible-playbook clean.yml -i local.hosts 
+```
+## Using OpenSDS
+OpenSDS features are accessible from a CLI or Dashboard
+Please refer [wiki](https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible) Section `How to test opensds cluster` for details of using OpenSDS features using CLI and OpenSDS Dashboard.
+## Known issues
+### 1. If port 80 is not free, installer will fail. 
+This is observed as multi-cloud (gelato) installation failure. Keystone require port 80 to be free and it will not start. Multi-cloud installation will wait for Keystone and timeout with failure.
+#### Workaround:
+Free up port 80 of the Host/VM for Keystone. 
+Example: If Apache2 service is using port 80, disable and stop this service with following steps
+```bash
```

Comment:
Have a doubt: If apache2 is stopped, will it affect any other service? or is it that we don't have any apache2 dependency

Reply:
"Apache2" service is given as example. For installer to work port 80 needs to be available